src/main/resources/calendars/i18n_nl.calendar

```diff
@@ -1,10 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <calendar id="NL" name="Netherlands Holidays 2018-2021" type="country">
-<!-- http://www.officeholidays.com/countries/netherlands/2012.asp -->
-<!-- http://calendar.retira.eu/public-holidays/netherlands/2012/ -->
+      <!-- http://www.officeholidays.com/countries/netherlands/2012.asp -->
+      <!-- http://calendar.retira.eu/public-holidays/netherlands/2012/ -->
+      <!-- https://www.vakantiedagennederland.nl/2023/ -->
 
       <date year="" month="1" date="1"><![CDATA[Nieuwjaarsdag]]></date>
       <date year="" month="4" date="27"><![CDATA[Koningsdag]]></date>
+      <date year="" month="5" date="5"><![CDATA[Bevrijdingsdag]]></date>
```

Comment:
Internets say that Liberation day is a holiday only once per five years [1]. If that's true, I recommend adding it as "NEUTRAL" day (add `type="NEUTRAL"` attribute)

1. https://www.officeholidays.com/holidays/netherlands/netherlands-liberation-day

Reply:
Thanks for noticing, you're correct, so I comitted the suggested change.
Apparently, liberation day is an official 'public holiday', but that does not mean that everyone has a day of from work. 
The latter depends on per company/sector. [2]
2. https://www.government.nl/topics/working-hours/question-and-answer/public-holidays-in-the-netherlands